docker/centos7-build

```diff
@@ -0,0 +1,128 @@
+FROM antaresrte/xpansion-centos7:1.1.1
+
+
+
+CMD ["/bin/bash"]
+
+RUN mkdir /workspace
+
+RUN echo "source /opt/rh/devtoolset-11/enable" >> /etc/bashrc
+SHELL ["/bin/bash", "--login", "-c"]
+
+
+RUN mkdir -p /workspace/vcpkg_cache
+ARG VCPKG_CACHE_DIR
+COPY $VCPKG_CACHE_DIR /workspace/
+
+
+ARG BRANCH=develop
+RUN cd /workspace && \
+    source /opt/rh/rh-git227/enable && \
+    git clone https://github.com/AntaresSimulatorTeam/antares-xpansion.git --branch $BRANCH && \
+    cd antares-xpansion && \
+    git submodule update --init vcpkg && ./vcpkg/bootstrap-vcpkg.sh -disableMetrics && \
+    export VCPKG_ROOT=/workspace/antares-xpansion/vcpkg
+
+
+
+
+RUN ORTOOLS_TAG=$(cat /workspace/antares-xpansion/antares-version.json | jq -r '."or-tools-rte"' ) && \
+    echo "ORTOOLS_TAG=$ORTOOLS_TAG" && \
+    URL_ORTOOLS=https://github.com/rte-france/or-tools-rte/releases/download/v$ORTOOLS_TAG/ortools_cxx_centos7_static_sirius.zip && \
+    mkdir -p ortools && cd ortools && \
+            wget -q -O ortools.zip $URL_ORTOOLS && \
+            unzip -q ortools.zip && \
+            rm ortools.zip
+
+RUN ANTARES_VERSION=$(cat /workspace/antares-xpansion/antares-version.json | jq -r '."antares_version"' ) && \
+    echo "ANTARES_VERSION=$ANTARES_VERSION" && \
+    mkdir -p deps && \
+    URL_ANTARES=https://github.com/AntaresSimulatorTeam/Antares_Simulator/releases/download/v$ANTARES_VERSION/antares-${ANTARES_VERSION}-CentOS-7.9.2009.tar.gz && \
+    wget $URL_ANTARES && \
+    tar -xvf antares-${ANTARES_VERSION}-CentOS-7.9.2009.tar.gz -C deps --strip-components=1 &&\
+    rm -rf antares-${ANTARES_VERSION}-CentOS-7.9.2009.tar.gz
+
+WORKDIR /workspace/antares-xpansion
+
+RUN pip3 install --upgrade pip && \
+    pip3 install wheel && \
+    pip3 install -r requirements-tests.txt && \
+    pip3 install -r requirements-ui.txt
```

Comment:
pip [accepts multiple arguments](https://stackoverflow.com/a/36744306)

```suggestion
    pip3 install -r requirements-tests.txt -r requirements-ui.txt
```

Reply:
good to know